creature_attributes.cpp

```diff
@@ -362,6 +363,29 @@ bool CreatureAttributes::canEquip() const {
   return !cantEquip;
 }
 
+static bool isInnateEffect(LastingEffect effect, CreatureId creatureId) {
```

Comment:
I prefer to show all effects, even innate. Btw, some creature receive permanent effects shortly after being created, so this code won't work in that case.